api/database/migrations/2022_11_25_232602_would_accept_temporary_to_position_duration.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+use Database\Helpers\ApiEnums;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Support\Facades\DB;
+
+class WouldAcceptTemporaryToPositionDuration extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
```

Comment:
I'd recommend actually removing the `would_accept_temporary` column after you've mapped the data across.  You can leave that deprecated column as a calculated field on the model dervied from position_duration.  That will make removing the field eventually much easier after the deprecation period.

Reply:
I will ask Tristan

Reply:
@tristan-orourke 
For the acceptance criteria task of deprecating wouldAcceptTemporary without totally breaking it, should I do this recommendation?
Or stick with what I have done? (keep it on the database, mark as deprecated wherever possible and remove from Schema otherwise)

Reply:
I was originally thinking of it exactly like you've implemented it @vd1992, but I like Peter's recommendation here. It may be safer to keep the database restricted to a single source of truth. We can keep the graphql wouldAcceptAttribute working by defining a [Laravel accessor](https://laravel.com/docs/8.x/eloquent-mutators#defining-an-accessor) for it which reverses the calculation you did in this migration.

Reply:
excellent, thanks!
I wanted to double check before dropping anything from the tables

Reply:
done in  https://github.com/GCTC-NTGC/gc-digital-talent/pull/4956/commits/a7fda4ba5843ef8e7e50732bd69e86f93442a572  and  https://github.com/GCTC-NTGC/gc-digital-talent/pull/4956/commits/7cf989b7a27ead986e991d47021457d5fb8c1f1b